src/cluster/serializer/binary-serialization-format/Plugin.cc

```diff
@@ -0,0 +1,19 @@
+#include "zeek/cluster/serializer/binary-serialization-format/Plugin.h"
+
+#include "zeek/cluster/Component.h"
+#include "zeek/cluster/serializer/binary-serialization-format/Serializer.h"
+
+
+using namespace zeek::plugin::Zeek_Binary_Serializer;
+using namespace zeek::cluster;
+
+zeek::plugin::Configuration Plugin::Configure() {
+    AddComponent(new LogSerializerComponent("ZEEK_BIN_V1", []() -> std::unique_ptr<LogSerializer> {
+        return std::make_unique<cluster::detail::BinarySerializationFormatLogSerializer>();
+    }));
+
+    zeek::plugin::Configuration config;
+    config.name = "Zeek::Binary_Serializer";
```

Comment:
IMO that's okay. There's other plugin names like `Zeek::MySQL`, `Zeek::File`, ` Zeek::Root` where it might not be clear that they provide. One can introspect with `zeek -NN` to get the components a plugin actually provides.

I could also see event serialization being built with Zeek's serialization format if we really wanted to (or log serialization with broker's format).